js/src/launchAsync.ts

```diff
@@ -116,7 +116,11 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             }
         };
 
-        const parent = options.parent ? options.parent : document.body;
+        options.parent = options.parent === null ? document.body : typeof(options.parent) !== 'undefined' ? options.parent : document.body;
+        if (!document.contains(options.parent)) {
```

Comment:
should combine this statement with above

const parent = options.parent && document.contains(options.parent) ? options.parent : document.body;

Reply:
done as suggested by it is assigned to options.parent and then to parent variable, both are used later and need to have same value